src/commands/balance.ts

```diff
@@ -0,0 +1,33 @@
+import { CommandInteraction } from 'discord.js';
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { Category } from 'types/applicationCommand';
+export async function run(
+    client: FluorineClient,
+    interaction: CommandInteraction
+) {
+    const balance = await client.economy.get(
+        interaction.user.id,
+        interaction.guild.id
+    );
+    const embed = new Embed(client, interaction.locale)
+        .setLocaleTitle('BALANCE')
+        .addLocaleField({
+            name: 'BALANCE_WALLET',
+            value: `${balance.wallet.toString()} ${await client.economy.getCurrency(
+                interaction.guildId
+            )}`
+        })
+        .addLocaleField({
+            name: 'BALANCE_BANK',
+            value: `${balance.bank.toString()} ${await client.economy.getCurrency(
+                interaction.guildId
+            )}`
+        });
+    interaction.reply({ embeds: [embed], ephemeral: true });
```

Comment:
```suggestion
    const currency = await client.economy.getCurrency(interaction.guildId);
    const embed = new Embed(client, interaction.locale)
        .setLocaleTitle('BALANCE')
        .addLocaleField({
            name: 'BALANCE_WALLET',
            value: `${balance.wallet} ${currency}`
        })
        .addLocaleField({
            name: 'BALANCE_BANK',
            value: `${balance.bank} ${currency}`
        });
    interaction.reply({ embeds: [embed], ephemeral: true });
```
awaiting the same statement twice will just make it 2x slower